msbuild/Messaging/Xamarin.Messaging.Build/TaskRunner.cs

```diff
@@ -17,14 +17,16 @@ internal TaskRunner (ITaskSerializer serializer)
 		{
 			this.serializer = serializer;
 
-			var dotnetPath = Path.Combine (MessagingContext.GetXmaPath (), "SDKs", "dotnet", "dotnet");
+			var sdkRootPath = Path.Combine (MessagingContext.GetXmaPath (), "SDKs");
+			var dotnetPath = Path.Combine (sdkRootPath, "dotnet", "dotnet");
 
 			//In case the XMA dotnet has not been installed yet
-			if (!File.Exists (dotnetPath)) {
+			if (File.Exists (dotnetPath)) {
+				Environment.SetEnvironmentVariable ("DOTNET_XMA_HOME", Path.Combine (sdkRootPath, ".home"));
```

Comment:
What happens if you just set HOME here? If that works, I don't think you need the rest of the changes.

Reply:
I could avoid the DOTNET_XMA_HOME and use HOME instead but the environment change on the base tasks would be needed anyways

Reply:
That's weird, if you set an environment variable for the current process, all child processes should inherit that variable

Reply:
@rolfbjarne It works, I just updated the PR by changing only the Build agent and setting HOME. I was verifying the wrong bits